src/views/UnitSimilarityMatrix/MatrixWidget.tsx

```diff
@@ -3,7 +3,8 @@ import { FunctionComponent, useCallback, useMemo, useState } from "react";
 import { AffineTransform, applyAffineTransform, applyAffineTransformInv, createAffineTransform, identityAffineTransform, inverseAffineTransform, multAffineTransforms } from "./AffineTransform";
 
 type Props = {
-    unitIds: (number | string)[]
+    unitIds1: (number | string)[]
+    unitIds2: (number | string)[]
```

Comment:
MatrixWidget has been slightly modified to allow two different sets of units... one for the columns and one for the rows. This is necessary for the confusion matrix.